Update SDK according to `2.0.0-pre-rc.14` (#153)
hyperledger/iroha-javascript: .changeset/thirty-adults-perform.md

---
'@iroha2/crypto-target-bundler': minor
'@iroha2/crypto-target-node': minor
'@iroha2/crypto-target-web': minor
'@iroha2/crypto-core': minor
---

> **refactor**: respect new `Algorithm` type & codec from `data-model` and combine it with the crypto's simple `Algorithm` type with is simply a string. Add `Algorithm.toDataModel` and `Algorithm.fromDataModel` methods.

Reviewer: ```suggestion
**refactor**: combine new `Algorithm` type and codec from `data-model` with the crypto's `Algorithm` type, which is simply a string. Add `Algorithm.toDataModel` and `Algorithm.fromDataModel` methods.
```

```suggestion
**refactor**: introduce new `Algorithm` type and codec from `data-model`. Add `Algorithm.toDataModel` and `Algorithm.fromDataModel` methods.
```
